provider/source/tests/data/cldr/cldr-units-full/main/ar/units.json

```diff
@@ -1920,14 +1920,8 @@
             "unitPattern-count-other": "{0} ربع غالون إمبراطوري"
           },
           "speed-light-speed": {
-            "gender": "feminine",
-            "displayName": "سرعة الضوء",
-            "unitPattern-count-zero": "سرعة الضوء {0}",
-            "unitPattern-count-one": "سرعة الضوء {0}",
-            "unitPattern-count-two": "سرعة الضوء {0}",
-            "unitPattern-count-few": "سرعة الضوء {0}",
-            "unitPattern-count-many": "سرعة الضوء {0}",
-            "unitPattern-count-other": "سرعة الضوء {0}"
+            "displayName": "light",
```

Comment:
light speed broken in many languages @sffc @srl295

Reply:
Confirmed that this is apparently WAI (this is a new item, and CLDR rolled back the localized data which was translated incorrectly across the board and they didn't have time to replace it with new data)